webapp-client/src/App.js

```diff
@@ -11,12 +11,10 @@ export default class App extends React.Component {
     // hard-coded for now, will be fetched from db
     this.defaultCategory = 'All';
     this.state = {
-      // list of category names for now, will be list of category ids later 
-      categories: [this.defaultCategory],
+      // categoryID of the current category
       currentCategory: this.defaultCategory,
```

Comment:
Is this part going to change when you merge in react-get-categoryList (upper case L)?